### PR TITLE
PP-927 as a user i want to pay using the terminal with ali pay and we chat

### PR DESCRIPTION
**Pull request by ioannis-kody**
As part of PP-927: Include QR payment fields and functionality

### CODE REVIEW

**Review by sishbi — COMMENTED** [review]
No README.md changes?

File: samples/php8/ecom-server/public/terminal_payment_form.php [hunk 1]
```diff
@@ -84,22 +91,65 @@ function generateRandomOrderId($length = 8) {
...
+        <label for="enable_payment_method">Enable payment method: </label>
+        <input type="checkbox" id="enable_payment_method" name="enable_payment_method" onchange="valueChanged()">
+    </div>
+
+    <div id="payment_method_fields" style="display: none; margin-bottom: 20px;
+    padding: 20px; border: 1px solid #ccc; border-radius: 4px">
+        <label for="payment_method_type">Payment method type:</label>
+        <select id="payment_method_type" name="payment_method_type">
+            <option value="CARD">Card</option>
+            <option value="ALIPAY">AliPay</option>
```
Comment (kody-joao) [inline samples/php8/ecom-server/public/terminal_payment_form.php:104]:
```suggestion
            <option value="ALIPAY">AliPay+</option>
```

File: samples/php8/ecom-server/public/terminal_payment_form.php [hunk 2]
```diff
@@ -84,22 +91,65 @@ function generateRandomOrderId($length = 8) {
...
+            <option value="WECHAT">WeChat</option>
+        </select>
+
+        <div style="display: flex; align-items: flex-end; margin-top:10px;">
+            <label for="activate_qr_code_scanner">Activate QR code scanner: </label>
```
Comment (kody-joao) [inline samples/php8/ecom-server/public/terminal_payment_form.php:109]: This is not built yet on terminal, let's update documentation only when it's ready

**Review by kody-joao — CHANGES_REQUESTED** [review]
Please remove the scanner function since that is not going to be released next on terminals

**Comment by kody-joao** [comment]
But maybe we keep it on a branch so that Harish can use it for testing the terminal development

**Review by kody-joao — APPROVED** [review]

**Pull request merged**